tools/UpdatePaths/Scripts/shuttle_engines.txt

```diff
@@ -0,0 +1,8 @@
+/obj/structure/shuttle/engine/@SUBTYPES : /obj/machinery/power/shuttle_engine/@SUBTYPES{@OLD}
```

Comment:
update the name of this file to have the number of this PR so it sorts correctly?